homeassistant/components/rflink.py

```diff
@@ -492,8 +492,7 @@ def cancel_queued_send_commands(self):
             # Rflink protocol/transport handles asynchronous writing of buffer
             # to serial/tcp device. Does not wait for command send
             # confirmation.
-            self.hass.async_create_task(self._protocol.send_command(
-                self._device_id, cmd))
+            self._protocol.send_command(self._device_id, cmd)
```

Comment:
are you sure about that ? On line 489 it's awaited and since it was passed to create_task, I would expect it to be a coroutine function.